### ISSUE TITLE
filter not working

### ISSUE THREAD

**Issue by lloricode** [rappasoft/laravel-livewire-tables#352]
from `v1.7.1` to `v1.10.0`

Filter is not interactive

![Screenshot from 2021-06-21 07-36-28](https://user-images.githubusercontent.com/8251344/122691624-75824900-d263-11eb-891e-4d12f862ebf8.png)

only config file are published with content:
```php
<?php

return [
    /**
     * Options: tailwind | bootstrap-4 | bootstrap-5.
     */
    'theme' => 'bootstrap-4',
];

```

**Comment by lloricode** [comment]
but this is started when using `1.8.0`

**Comment by rappasoft** [comment]
Can't replicate. This is BS4 on 1.10:


https://user-images.githubusercontent.com/6026504/122697942-54ded280-d214-11eb-913c-1531b3c7f818.mov

**Comment by lloricode** [comment]
> Can't replicate. This is BS4 on 1.10:
> 
>  Screen.Recording.2021-06-20.at.22.09.54.mov

thanks for the response, we currently investigate if this in our project issue

**Comment by rappasoft** [comment]
Make sure you clear the view cache as we made changes to the way those dropdowns work.

**Comment by juliomotol** [comment]
Please make sure you have `alpinejs` installed since prior to `^1.9`, the filter dropdown no longer uses bootstrap's scripts

**Comment by rappasoft** [comment]
Alpine has been a prerequisite since v1. Top line: https://github.com/rappasoft/laravel-livewire-tables/wiki/Installation-&-Configuration

**Comment by lloricode** [comment]
thank for the help, I will close this issue if we make it on our side

**Comment by lloricode** [comment]
fixed with #354
